diff -ruN mesa-21.3.7/src/gallium/targets/dri/meson.build mesa-21.3.7_mod/src/gallium/targets/dri/meson.build
--- mesa-21.3.7/src/gallium/targets/dri/meson.build	2022-02-23 19:35:29.000000000 +0100
+++ mesa-21.3.7_mod/src/gallium/targets/dri/meson.build	2022-03-26 15:07:03.246720444 +0100
@@ -85,6 +85,7 @@
                'pl111_dri.so',
                'repaper_dri.so',
                'rockchip_dri.so',
+               'rockchip-ebc_dri.so',
                'st7586_dri.so',
                'st7735r_dri.so',
                'stm_dri.so',
diff -ruN mesa-21.3.7/src/gallium/targets/dri/target.c mesa-21.3.7_mod/src/gallium/targets/dri/target.c
--- mesa-21.3.7/src/gallium/targets/dri/target.c	2022-02-23 19:35:29.000000000 +0100
+++ mesa-21.3.7_mod/src/gallium/targets/dri/target.c	2022-03-26 15:08:32.005868718 +0100
@@ -114,6 +114,7 @@
 DEFINE_LOADER_DRM_ENTRYPOINT(pl111)
 DEFINE_LOADER_DRM_ENTRYPOINT(repaper)
 DEFINE_LOADER_DRM_ENTRYPOINT(rockchip)
+DEFINE_LOADER_DRM_ENTRYPOINT(rockchip_ebc)
 DEFINE_LOADER_DRM_ENTRYPOINT(st7586)
 DEFINE_LOADER_DRM_ENTRYPOINT(st7735r)
 DEFINE_LOADER_DRM_ENTRYPOINT(stm)
